databus/src/main/java/com/bazaarvoice/emodb/databus/core/DefaultFanout.java

```diff
@@ -105,6 +113,11 @@ public DefaultFanout(String name,
         _lastLagStopwatch = Stopwatch.createStarted(ClockTicker.getTicker(clock));
         _clock = clock;
         ServiceFailureListener.listenTo(this, metricRegistry);
+
+        _fanoutPool = Executors.newFixedThreadPool(
+            8,
```

Comment:
If there is a chance that the size of the thread pool can affect performance, does it make sense to make this value configurable?

Reply:
yeah, I thought about that... It's a project preference. For my projects, I package the config with the code, so it doesn't typically matter to me whether a setting is in config or code, and code is better since it's right there for all to see.

You may have a different preference, which is fine.

I do think that it's not so much about finding the "best" setting and just finding one that buys you some extra time before swapping out all this code for a more standard distributed queue implementation.

Reply:
weird, this comment showed up twice. see my other answer.